packages/next/src/server/config-shared.ts

```diff
@@ -286,6 +286,8 @@ export interface ExperimentalConfig {
   extensionAlias?: Record<string, any>
   allowedRevalidateHeaderKeys?: string[]
   fetchCacheKeyPrefix?: string
+  imgOptTimeoutInSeconds?: number
```

Comment:
Worth adding to the defaults to `defaultConfig` so that we can warn when it's enabled:
https://github.com/vercel/next.js/blob/242faf08fbc3cffdb9b9d42605a5e4e6f3410077/packages/next/src/server/config-shared.ts#L1042